config/config.go

```diff
@@ -334,6 +340,8 @@ type ScrapeConfig struct {
 	ServersetSDConfigs []*ServersetSDConfig `yaml:"serverset_sd_configs,omitempty"`
 	// MarathonSDConfigs is a list of Marathon service discovery configurations.
 	MarathonSDConfigs []*MarathonSDConfig `yaml:"marathon_sd_configs,omitempty"`
+	// MarathonSDConfigs is a list of Marathon service discovery configurations.
+	MesosSDConfigs []*MesosSDConfig `yaml:"mesos_sd_configs,omitempty"`
```

Comment:
No, we do not use a system like that. But I thought about the naming too. Would `MesosSlaveSDConfigs` be fine (along with renaming it in all other places)?